### PR TITLE
Add libertas.eth

### PR DESCRIPTION
**Pull request by TheLibertasProject**
#### Hi! What is your PR about?

- [x] Add images for your space
- [x] Add custom domain for your space
- [ ] Add alias for migrate your space
- [x] Add or edit a skin
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: skins/uniswap.scss [hunk 1]
```diff
@@ -9,6 +9,7 @@
   --block-bg: transparent;
 }
 
+// dark theme
```
Comment (gokaiorg) [inline skins/uniswap.scss:12]: Please remove this comment. 
```suggestion
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]
There is a conflict in your PR since `domains.json` file has been updated, can you resolve it?

**Comment by TheLibertasProject** [comment]
I will resubmit a new one. Domains.json is still incorrect.

File: skins/uniswap.scss (hunk 1 above)
Comment (TheLibertasProject) [reply skins/uniswap.scss]: Done

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**